wsmaster/che-core-api-user/src/main/java/org/eclipse/che/api/user/server/UserManager.java

```diff
@@ -38,7 +42,7 @@
  * @author Max Shaposhnik (mshaposhnik@codenvy.com)
  */
 public class UserManager {
-
+    List<String> reservedNames = new ArrayList<>();
```

Comment:
missing java modifier (probably private)
also we don't need to initialize variable to a list as in constructor we assign the value